### ISSUE TITLE
Long line without spaces in Alert doesn't wrap

### ISSUE THREAD

**Issue by benag** [twbs/bootstrap#16295]
Solved when "," is replaced with " , ".

**Comment by benag** [comment]
see: http://jsfiddle.net/4wv0su4d/

**Comment by twbs-lmvtfy** [comment]
Hi @benag!

You appear to have posted a live example (http://fiddle.jshell.net/4wv0su4d/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 66, column 1: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 67, column 1: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 12, column 3: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.2.0
* line 13, column 3: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.2.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Approximate duplicate of #13691.
